class Fault():
    def __init__(self):
        pass
    